Classes/Issues/Comments/Reactions/ReactionContent+ReactionType.swift

```diff
@@ -19,4 +19,16 @@ extension ReactionContent {
         case .thumbsDown: return "👎"
         }
     }
+    
+    /// User readable name of the current reaction type
+    var localizedString: String {
+        switch self {
+        case .confused: return NSLocalizedString("confused", comment: "")
+        case .heart: return NSLocalizedString("heart", comment: "")
+        case .hooray: return NSLocalizedString("hooray", comment: "")
+        case .laugh: return NSLocalizedString("laugh", comment: "")
+        case .thumbsUp: return NSLocalizedString("thumbs up", comment: "")
+        case .thumbsDown: return NSLocalizedString("thumbs down", comment: "")
+        }
+    }
```

Comment:
See note about dropping the emoji name from the tooltip, we can probably get rid of these.